crates/sui-deepbook-indexer/src/sui_deepbook_indexer.rs

```diff
@@ -0,0 +1,806 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
we should move `storage.rs`  or at least the progressSavingPolicy thing to `sui_indexer_builder` crate to avoid duplicated code